test_spam_no_permission depends on .env (MULTI_AUTH_ENABLED)

No tests should depend on the personal `.env` file. If we allow that to matter, tests become unpredictable. 

Here's the error and I put in some additional print lines:
```
Traceback (most recent call last):
  File "/app/kuma/wiki/tests/test_views_akismet.py", line 137, in test_spam_no_permission
    assert response["Location"].endswith("users/signin?next={}".format(url)), response[
AssertionError: /en-US/users/account/signup-landing?next=/en-US/docs/submit_akismet_spam
assert False
 +  where False = <built-in method endswith of str object at 0x7ff297a9b3b0>('users/signin?next=/en-US/docs/submit_akismet_spam')
 +    where <built-in method endswith of str object at 0x7ff297a9b3b0> = '/en-US/users/account/signup-landing?next=/en-US/docs/submit_akismet_spam'.endswith
 +    and   'users/signin?next=/en-US/docs/submit_akismet_spam' = <built-in method format of str object at 0x7ff2a602ce40>('/en-US/docs/submit_akismet_spam')
 +      where <built-in method format of str object at 0x7ff2a602ce40> = 'users/signin?next={}'.format
----------------------------------------------------------------------------------------------- Captured stdout call ------------------------------------------------------------------------------------------------
LOCALTION: /en-US/users/account/signup-landing?next=/en-US/docs/submit_akismet_spam
EXPECTED 'users/signin?next=/en-US/docs/submit_akismet_spam'
```

--- a/kuma/wiki/tests/test_views_akismet.py
+++ b/kuma/wiki/tests/test_views_akismet.py
@@ -132,7 +132,7 @@ def test_spam_no_permission(
     )
     # Redirects to login page when without permission.
     assert response.status_code == 302
-    assert response["Location"].endswith("users/signin?next={}".format(url))
+    assert response["Location"].endswith(f"{reverse(settings.LOGIN_URL)}?next={url}")
     assert_no_cache_header(response)
 
     # No RevisionAkismetSubmission record should exist.

